### DISCUSSION TITLE
Overview for all (default) keybindings, and commands and such?

### DISCUSSION THREAD

**Discussion by Snadgh** [epwalsh/obsidian.nvim#697] [Q&A] [answered]
Hi! This might be a stupid question and I might be just overlooking - but is there like an overview of all of the commands and keybindings that are default to the program? I know about the list in the README.md file, but it seems like there is more to it than just that list - is that right?

**Comment by TheMeaningfulEngineer** [comment, marked as answer]
Partial answer.
If you do `:Obsidian<TAB>` it might print out all the functions.
I guess that depends on your config, but it's worth a try.

![image](https://github.com/user-attachments/assets/89db64f5-7ba8-4704-9bba-8bbe8e449986)
